server_admin/topics/users/create-user.adoc

```diff
@@ -1,19 +1,17 @@
-[[_create-new-user]]
-
-=== Creating New Users
-
-To create a user click on `Users` in the left menu bar.
-
-.Users
-image:{project_images}/users.png[]
-
-This menu option brings you to the user list page.  On the right side of the empty user list, you should see an `Add User`
-button.  Click that to start creating your new user.
-
-.Add User
-image:{project_images}/add-user.png[]
-
-The only required field is `Username`.  Click save.  This will bring you to the management page for your new user.
+[creat-new-users]
+= Creating new users
+
+You can use the menu options to create new users and update the user list page. 
```

Comment:
I am not sure why there is information other than creating users here. If this procedure covers multiple use cases, the time should reflect. I would add some information about why or needs to create users rather than a sentence that gives multiple options that are then not presented below.